l10n_it_ricevute_bancarie/models/riba.py

```diff
@@ -216,6 +226,11 @@ def action_cancel_draft(self):
             for line in riba_list.line_ids:
                 line.state = "draft"
 
+    def action_open_lines(self):
+        action = self.env.ref("l10n_it_riba.detail_riba_action").read()[0]
```

Comment:
Va modificato l'ID esterno in `l10n_it_ricevute_bancarie.detail_riba_action`

Reply:
grazie